README.md

```diff
@@ -172,12 +172,17 @@ If you are not creating or using an Application class throughout your project, a
 
 ```java
 Branch branch = Branch.getInstance(getApplicationContext());
-branch.initSession(new BranchReferralInitListener(){
+branch.initSession(new Branch.BranchUniversalReferralInitListener() {
```

Comment:
Because the BUO is not the only way to create links, we can't make this the default unfortunately : / You can add this as an option below the main JSONObject version, as we do in a couple other repos.